docker-compose/common-services/common-services.yaml

```diff
@@ -6,27 +6,84 @@ services:
     environment:
       - REFERENCE_SERVER_CONFIG_ENV=file:/config/reference-config.yaml
     depends_on:
-      - db
-      - kafka
+      - db_a
+      - kafka_a
     volumes:
       - .:/config
     ports:
       - "8081:8081" # reference-server
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  anchor_platform_server_a:
+    image: stellar/anchor-platform:edge
+    build:
+      context: ../../
+      dockerfile: integration-tests/docker-compose-configs/Dockerfile
+    command: "--sep-server"
+    environment:
+      - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
+      - SECRET_SEP10_SIGNING_SEED=${SECRET_SEP10_SIGNING_SEED?err}
+      - SECRET_SEP10_JWT_SECRET=secret
+      - SECRET_PLATFORM_API_AUTH_SECRET=myAnchorToPlatformSecret
+      - SECRET_CALLBACK_API_AUTH_SECRET=myPlatformToAnchorSecret
+      - SECRET_DATA_USERNAME=postgres
+      - SECRET_DATA_PASSWORD=password
+      - LOG_APPENDER=console_appender
+      - SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET=secret_sep24_interactive_url_jwt_secret
+      - SECRET_SEP24_MORE_INFO_URL_JWT_SECRET=secret_sep24_more_info_url_jwt_secret
+    volumes:
+      - .:/config
+    depends_on:
+      - kafka_a
+      - db_a
+      - reference_server_a
+    ports:
+      - "8080:8080"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  observer_a:
+    image: stellar/anchor-platform:edge
+    build:
+      context: ../../
+      dockerfile: integration-tests/docker-compose-configs/Dockerfile
+    command: "--stellar-observer"
+    environment:
+      - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
+      - SECRET_SEP10_SIGNING_SEED=${SECRET_SEP10_SIGNING_SEED?err}
+      - SECRET_SEP10_JWT_SECRET=secret
+      - SECRET_PLATFORM_API_AUTH_SECRET=myAnchorToPlatformSecret
+      - SECRET_CALLBACK_API_AUTH_SECRET=myPlatformToAnchorSecret
+      - SECRET_DATA_USERNAME=postgres
+      - SECRET_DATA_PASSWORD=password
+      - LOG_APPENDER=console_appender
+    volumes:
+      # add mounts for the new config directory
+      - .:/config
+    depends_on:
+      - db_a
+      - kafka_a
+      - reference_server_a
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
 
   kafka_a:
     platform: linux/x86_64
     image: confluentinc/cp-kafka:6.1.9
     depends_on:
-      - zookeeper
+      - zookeeper_a
     ports:
       - "29092:29092"
     environment:
       KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper_a:2181
```

Comment:
Would this break the `sep24` and `sep31` docker-compose?

Reply:
Not sure if that would break the other docker-compose file. But anyway, I needed to update the services' names because of the dependencies. Sep24 docker-compose was expecting for a db_a to initialize, but it is the service name from the common-services file.